packages/app/src/app/overmind/namespaces/explore/actions.ts

```diff
@@ -23,30 +23,21 @@ export const pickSandbox: AsyncAction<{
   description: string;
 }> = async ({ state, actions, effects }, { id, title, description }) => {
   try {
-    if (!state.explore.popularSandboxes) {
-      return;
-    }
-
     const data = await effects.api.saveSandboxPick(id, title, description);
-    const popularSandbox = state.explore.popularSandboxes.sandboxes.find(
-      module => module.id === id
-    );
+    const popularSandbox = (
+      state.explore.popularSandboxes?.sandboxes || []
+    ).find(module => module.id === id);
 
-    if (!popularSandbox) {
-      effects.notificationToast.error(
-        'Could not pick Sandbox, please refresh and try again'
-      );
-      return;
+    if (popularSandbox) {
+      popularSandbox.picks = [
+        {
+          ...data,
+          // Why are we doing this?
+          id: Math.random().toString(),
```

Comment:
I did that at the time but honestly I don't remeber why I think it was for all picks to have an id

Reply:
Ah, cool, thanks @SaraVieira :) 

Btw, I added a fix now so that it pushes to that array, not replacing it. To avoid it overriding existing picks... not sure how this works, but yeah, seems more correct :)